_scripts/toolbox-create

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/bash -e
 
 echo "## Creating website container..."
-toolbox create -c cockpit-website -r 33
+toolbox create -c cockpit-website -r 34
```

Comment:
Making the decision to lock the container to specific Fedora releases paid off. Ruby was upgraded to 3.x in Fedora 34 and it broke things, so keeping it on Fedora 33 for a while until investigating what happened was the right choice.